tools/db_crashtest.py

```diff
@@ -858,6 +858,9 @@ def finalize_and_sanitize(src_params):
     elif (dest_params.get("use_put_entity_one_in") > 1 and
         dest_params.get("use_timed_put_one_in") == 1):
         dest_params["use_timed_put_one_in"] = 3
+    # TODO: renable this when file ingestion is failed during wal lock.
+    if dest_params.get("lock_wal_one_in") != 0:
```

Comment:
In most configurations, lock_wal_one_in is always != 0. I'm not sure there are any configurations remaining with a chance of testing ingestion

Reply:
That's a good point! I updated this to randomly disable one of the two features so that they do not run together, but can both get tested.